frame/contracts/src/wasm/runtime.rs

```diff
@@ -300,7 +318,7 @@ define_env!(Env, <E: Ext>,
 	//
 	// - amount: How much gas is used.
 	gas(ctx, amount: u32) => {
-		charge_gas(&mut ctx.gas_meter, ctx.schedule, RuntimeToken::Explicit(amount))?;
+		charge_gas(&mut ctx.gas_meter, ctx.schedule, &mut ctx.special_trap, RuntimeToken::Explicit(amount))?;
```

Comment:
This (an a few other) line exceeds the soft limit of 100. It would be great if we wrapped this line, like

```rust
charge_gas(
    &mut ctx.gas_meter,
    ctx.schedule,
    &mut ctx.special_trap,
    RuntimeToken::Explicit(amount)
)?;
```

Reply:
I will look into the overlong lines.

Reply:
Removed the long lines I added / mofified.